Zappy_IA_Report/NN/Code/Tutorial4.py

```
@@ -0,0 +1,31 @@
+import tensorflow as td
+from tensorflow import keras
+import numpy as np
+
+data = keras.datasets.imdb
+
+(train_data, train_labels), (test_data, test_labels) = data.load_data(num_words=88000)
+
+word_index = data.get_word_index()
+
+word_index = {k:(v+3) for k, v in word_index.items()}
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)